diff --git a/androidApp/src/main/kotlin/gr/questweaver/android/MainApp.kt b/androidApp/src/main/kotlin/gr/questweaver/android/MainApp.kt
index e9dfa46..b78aab3 100644
--- a/androidApp/src/main/kotlin/gr/questweaver/android/MainApp.kt
+++ b/androidApp/src/main/kotlin/gr/questweaver/android/MainApp.kt
@@ -11,4 +11,4 @@ class MainApp : Application() {
             injectModules()
         }
     }
-}
\ No newline at end of file
+}
